README.md

```diff
@@ -172,6 +174,26 @@ As in the case of NIFs, CNodes compiled with Bundlex can be used like any other
 Similarly to CNodes Bundlex provides `Bundlex.Port` module for a little easier interacting with Ports.
 Please refer to the module's documentation to see how to use it.
 
+### Cross-compilation
+
+With proper setup, Bundlex can support cross-compilation. When using Nerves it should work out of the box. 
+
+Not relying on Nerves and using your own toolchain is also possible, although it wasn't tested. In this scenario, the following environment variables need to be set during compilation (when changing the target bundlex also needs to be recompiled):
+- `CROSSCOMPILE`
```

Comment:
lack of description

Reply:
because it just needs to be set, it's value is not used anywhere